### PR TITLE
Add `crypto_keys` field

### PR DESCRIPTION
**Pull request by nurdann**
Related to #3536. Choosing on `crypto_keys` field name.

### CODE REVIEW

**Review by nemacysts — COMMENTED** [review]
this looks fine as-is - but it might be worth to hold off on merging any changes for this feature until the CEP is signed off (e.g., I just made a comment/suggestion about what how this dict could look)

File: paasta_tools/cli/schemas/tron_schema.json
Comment (nemacysts) [inline paasta_tools/cli/schemas/tron_schema.json:223]: just fyi: we may need to update tron/task_processing to support crypto_keys

File: paasta_tools/cli/schemas/adhoc_schema.json
Comment (nemacysts) [inline paasta_tools/cli/schemas/adhoc_schema.json:68]: would it help to add a regex pattern to the schema for what the items should look like (e.g., the stuff in these arrays should match `[a-zA-Z]+`)?

**Review by nemacysts — APPROVED** [review]

File: paasta_tools/cli/schemas/tron_schema.json
Comment (nurdann) [reply paasta_tools/cli/schemas/tron_schema.json]: where would that need to happen?

Comment (nemacysts) [reply paasta_tools/cli/schemas/tron_schema.json]: for the payload that tron gets: probably somewhere around https://github.com/Yelp/paasta/blob/master/paasta_tools/tron_tools.py#L884

for processing in tron: probably something like https://github.com/Yelp/Tron/pull/822

and for task_processing: you'll likely want to pick up the work in https://github.com/Yelp/task_processing/pull/188/files


that said: if we're not expecting tronjobs to need this, it'll be a lot easier to just exclude this from the schema :p

**Review by nemacysts — APPROVED** [review]

File: paasta_tools/cli/schemas/tron_schema.json
Comment (nurdann) [reply paasta_tools/cli/schemas/tron_schema.json]: If the need arises, we can support direct authentication via AWS. So we don't need to make changes for tronjobs currently.

**Pull request merged**
